navigation/screens/home.tsx

```diff
@@ -17,9 +17,8 @@ import { getUserProfile } from "@/controllers/userController";
 /* eslint-disable */
 const map = require("../../assets/images/map.jpg");
 
-export default function MainScreen() {
+export default function MainScreen(bathroomID: string, username: string) {
```

Comment:
neither of these belong here. 

1. bathroomID: the button for each bathroom on the map will know it's bathroom ID and pass it to the facility-profile component when it navigates there. 
2. username: main screen does not allow the user to edit, so it probably doesn't need to know the user name or that the user is authenticated even